src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails.jsx

```diff
@@ -215,17 +218,63 @@ class GroupEventDetails extends React.Component {
   }
 }
 
-export {GroupEventDetails};
+class GroupEventDetailsEnvWrapper extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    selection: SentryTypes.GlobalSelection.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+  };
 
-export default withApi(
-  withOrganization(
-    withGlobalSelection(props => {
-      const {selection, ...otherProps} = props;
-      const environments = OrganizationEnvironmentsStore.getActive().filter(env =>
-        selection.environments.includes(env.name)
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Let's move this out of the constructor and define it in class properties... should the initial state values be = `OrganizationEnvironmentsStore.getActive()`?